x-pack/plugins/stack_connectors/server/connector_types/webhook/index.ts

```diff
@@ -54,6 +55,12 @@ const configSchemaProps = {
   }),
   headers: nullableType(HeadersSchema),
   hasAuth: schema.boolean({ defaultValue: true }),
+  authType: schema.string({ defaultValue: WebhookAuthType.Basic }),
```

Comment:
To avoid breaking changes this should be `schema.maybe(schema.string({ defaultValue: WebhookAuthType.Basic }))`. Also maybe we can narrow the type to only accept one of the `WebhookAuthType` and not any string.